src/client/index.ts

```diff
@@ -344,8 +367,16 @@ export async function create(options: NotificationOptions): Promise<Notification
         throw new Error('Invalid arguments passed to create: "date" must be a valid Date object');
     }
 
-    const response = await tryServiceDispatch(APITopic.CREATE_NOTIFICATION, {...options, date: options.date && options.date.valueOf()});
-    return {...response, date: new Date(response.date)};
+    if (options.expiry !== undefined && options.expiry !== null && !(options.expiry instanceof Date)) {
```

Comment:
Can't `options.expiry !== undefined && options.expiry !== null` be simplified to `options.expiry &&`?

Reply:
I had to stare at this a bit, but if someone set the date as `false` or `0`, with that change the error wouldn't get thrown